lib/firebase_token_auth/version.rb

```diff
@@ -1,3 +1,3 @@
 module FirebaseTokenAuth
-  VERSION = '1.3.0'.freeze
```

Comment:
Please exclude this change.
I'll change version number by myself after add some test code.